aws_lambda_powertools/event_handler/util.py

```diff
@@ -40,3 +69,30 @@ def extract_origin_header(resolver_headers: Dict[str, Any]):
         return resolved_header[0]
 
     return resolved_header
+
+
+def validate_openapi_security_parameters(
```

Comment:
```suggestion
def _validate_openapi_security_parameters(
```

make it private to avoid breakage, as we should move these type of validations into a Schema Validator (missing?) once we have one

Reply:
Great catch!